client/src/featureform/get.py

```diff
@@ -58,154 +39,118 @@ def get_resource_info(stub, resource_type, name):
         "training-set": stub.GetTrainingSets,
         "model": stub.GetModels
     }
-
     searchName = metadata_pb2.Name(name=name)
     try:
         for x in stub_get_functions[resource_type](iter([searchName])):
-            format_rows([("NAME: ", x.name),
-            ("STATUS: ", x.status.Status._enum_type.values[x.status.status].name)])
-            format_pg("VARIANTS:")
-            format_rows(x.default_variant, 'default')
-            for v in x.variants:
-                if v != x.default_variant:
-                    format_rows(v, '')
-            format_pg()
-            return x
+            return StubResource(x)
     except grpc._channel._MultiThreadedRendezvous:
         print(f"{resource_type} not found.")
 
+class StubResource:
```

Comment:
What is the purpose of this? It seems like its just taking a formatted print and making it not formatted